src/context_free.ml

```diff
@@ -459,9 +476,11 @@ class map_top_down ?(expect_mismatch_handler = Expect_mismatch_handler.nop)
 
     method! core_type base_ctxt x =
       map_node EC.core_type core_type super#core_type x.ptyp_loc base_ctxt x
+        ~embed_errors
```

Comment:
The diff can be made quite a lot smaller by redefining `map_node` and similar functions with a fixed value for `embed_errors`. Similar to what is done [here](https://github.com/ocaml-ppx/ppxlib/pull/453/files#diff-f74171f29027fe3de9bf4383816c1a4599ab4851ecf6181abb5f41fbd39b06b9R468-R469) with the `hook` value!

Reply:
Thank you Sir, this link directs back to this pr.